pkg/storage/mvcc.go

```diff
@@ -7318,7 +7323,23 @@ func mvccExportToWriter(
 	if err != nil {
 		return kvpb.BulkOpSummary{}, ExportRequestResumeInfo{}, err
 	}
-	defer iter.Close()
+	defer func() {
+		if opts.ScanStats != nil {
+			recordIteratorStats(iter, opts.ScanStats)
+			opts.ScanStats.NumScans = 1
```

Comment:
What's the reason for hardcoding it to 1 instead of incrementing? `recordIteratorStats` will add to existing values, but NumScan is overwritten. I don't think it makes any difference for cmd_export, but it would be nice to have it consistent if we change it to pass non 0 values.